File: Back-end/src/HorizonEvents.Domain/Event.cs
using System;
using System.Collections.Generic;

namespace HorizonEvents.Domain
{
    public class Event
    {
        public int Id { get; set;}
        public string Local { get; set; }
        public DateTime EventDate { get; set; }
        public string Theme { get; set; }
        public int NumPeople { get; set; }
        public string ImageURL { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public IEnumerable<Batch> Batches { get; set; }
        public IEnumerable<SocialMedia> SocialMedias { get; set; }
        public IEnumerable<SpeakerEvent> SpeakerEvents { get; set; }

    }
}

File: Front-End/HorizonEvents-App/src/app/models/Event.ts
import { Batch } from "./Batch";
import { SocialMedia } from "./SocialMedia";
import { Speaker } from "./Speaker";

export interface Event{
 id: number;
 local: string;
 eventDate?: Date;
 theme: string;
 numPeople: number;
 imageURL: string;
 phoneNumber: string;
 email: string;
 batches: Batch[];
 socialMedias: SocialMedia[];
 speakerEvents: Speaker[];
}
